src/System.Diagnostics.Debug/tests/DebugTests.cs

```diff
@@ -22,14 +25,14 @@ public void Asserts()
             VerifyAssert(() => { Debug.Assert(false, "assert passed", "nothing is wrong {0} {1}", 'a', 'b'); }, "assert passed", "nothing is wrong a b");      
         }
 
-        [Fact]
+        [ConditionalFact(nameof(IsNotDesktopJob))]
```

Comment:
These APIs do exist on desktop and I would expect them to behave the same. I could imagine cases that are different when running on debug builds of the product but that should be a different pivot. 

In any case if we need to condition based on desktop we should use the TestTFM approach I called out elsewhere.